lib/philomena_web/controllers/api/json/filter_controller.ex

```diff
@@ -0,0 +1,41 @@
+defmodule PhilomenaWeb.Api.Json.FilterController do
+  use PhilomenaWeb, :controller
+
+  alias PhilomenaWeb.FilterJson
+  alias Philomena.Filters.Filter
+  alias Philomena.Repo
+  import Ecto.Query
+
+  plug PhilomenaWeb.RecodeParameterPlug, [name: "id"] when action in [:show]
```

Comment:
This plug is used for tag slug breakage, it isn't necessary for integers like filter ids.